docs/herbarium/getting-started.md

```diff
@@ -0,0 +1,40 @@
+---
+id: gettingStarted
+title: Getting Started with Herbarium
+sidebar_label: Getting Started
+slug: /herbarium/getting-started
+---
+
+### Herbarium
+
+Herbarium is a centralized and standardized repository and discovery service for Herbs objects (entities, use cases, repositories) allowing glues (ex: Herbs Shelf, herbs2rest, herbs2gql, herbs2knex, etc) to access, explore and find these objects.
+
+### Installing
+
+```javascript
+$ npm install @herbsjs/herbarium
+```
+
+### Initializing
+
+```javascript
+const { herbarium } = require('@herbs/herbarium')
+herbarium.requireAll(options)
+```
+
+This will require all entities, use cases and repositories files.
```

Comment:
Suggested text:
This will find and `require()` all entities, use cases and repositories files. If these files are using Herbarium, it will load all the objects to be used later.